packages/@react-aria/list/docs/useList.mdx

```diff
@@ -0,0 +1,439 @@
+for assistive technology. Each of these hooks returns props to be spread onto the appropriate HTML element.
+
+State is managed by the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} />
+hook from `@react-stately/list`. The state object should be passed as an option to each of the above hooks where applicable.
+
+Note that an `aria-label` or `aria-labelledby` must be passed to the list to identify the element to assistive technology.
+
+## State management
+
+`useList` requires knowledge of the rows in the list in order to handle keyboard
+navigation and other interactions. It does this using
+the <TypeLink links={collectionsDocs.links} type={collectionsDocs.exports.Collection} />
+interface, which is a generic interface to access sequential unique keyed data. You can
+implement this interface yourself, e.g. by using a prop to pass a list of item objects,
+but <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} /> from
+`@react-stately/list` implements a JSX based interface for building collections instead.
+See [Collection Components](/react-stately/collections.html) for more information,
+and [Collection Interface](/react-stately/Collection.html) for internal details.
+
+See the examples in the [usage](#usage) section below for details on how to use these components.
+
+In addition, <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} />
+manages the state necessary for multiple selection and exposes
+a <TypeLink links={selectionDocs.links} type={selectionDocs.exports.SelectionManager} />,
+which makes use of the collection to provide an interface to update the selection state.
+For more information, see [Selection](/react-stately/selection.html).
+
+## Example
+
+Lists are [collection components](../react-stately/collections.html) that include rows as child elements.
+In this example, we'll use the standard HTML unordered list elements along with hooks from React
+Aria for each child. You may also use other elements like `<div>` to render these components as appropriate.
+We'll walk through creating the list container and list item, then add some additional behavior such as selection.
+
+The <TypeLink links={docs.links} type={docs.exports.useList} /> hook will be used to render the outer most list element. It uses
+the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} /> hook to construct the list's collection of rows,
+and manage state such as the focused row and row selection. We'll use the collection to iterate through
+the rows of the List and render the relevant components, which we'll define below.
+
+You may notice the extra `<div>` with `gridCellProps` in our example. This is needed because we are following the [ARIA grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/).
```

Comment:
```suggestion
You may notice the extra `<div>` with `gridCellProps` in our example. This is needed because we are following the [ARIA grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/), which does not allow rows without any child `gridcell` elements.
```